### ISSUE TITLE
Python PIP is missing

### ISSUE THREAD

**Issue by stdevel** [timorunge/ansible-freeipa-server#8]
By default, PIP is not installed on EL7 and therefore the role fails:
```
    TASK [timorunge.freeipa_server : Install FreeIPA server pip dependencies] ******
    FAILED - RETRYING: Install FreeIPA server pip dependencies (3 retries left).
    FAILED - RETRYING: Install FreeIPA server pip dependencies (2 retries left).
    FAILED - RETRYING: Install FreeIPA server pip dependencies (1 retries left).
fatal: [centos-7]: FAILED! => {"attempts": 3, "changed": false, "msg": "Unable to find any of pip2, pip to use.  pip needs to be installed."}
```

My suggestion is to add ``python2-pip`` to ``vars/RedHat.yml`` as Python 2.x is the system-wide default for EL7, Fedora 27 and 28.

I created the PR #9 for this.

**Comment by timorunge** [comment]
Thank you @stdevel! 

I've merged #9. :)

**Comment by stdevel** [comment]
You're welcome, thanks for merging the code! :)
